packages/fxa-auth-server/bin/mjml-server.ts

```diff
@@ -5,9 +5,9 @@
 import http from 'http';
 import stream from 'stream';
 import net from 'net';
-import { render } from '../lib/senders/emails/renderer';
+import { renderWithOptionalLayout } from '../lib/senders/emails/renderer';
```

Comment:
I think you will also need to update https://github.com/mozilla/fxa/blob/8a5e52f0d05892b0bcdb1d44753e5d37bc856e46/packages/fxa-auth-server/lib/senders/emails/render-to-disk.ts#L18